dawai-framework/packages/dawai-protocol-core/src/lib/interfaces.ts

```
@@ -1,16 +1,31 @@
 export interface IMCPMethod {
-  // Placeholder for MCP method structure
+  name: string;
+  description: string;
+  parameters: { [key: string]: string }; // Basic type hints, e.g., { "arg1": "number", "arg2": "string" }
+}
+
+// For A2A interfaces, as the PRD provides no specific structure beyond placeholders,
+// define them as empty interfaces for now.
+export interface IA2AMessage {
+  // A2A specific message structure (currently undefined in PRD example for Epic 2)
 }
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)